[Logs UI] Update log threshold alert docs (#155)
elastic/observability-docs: docs/en/observability/logs-threshold-alert.asciidoc

To determine how many log entries would match each part of your configuration, you can view a chart preview for each condition. This is useful for determining how many log entries would match each part of your configuration. When a group by is set, the chart displays a bar per group. To view the preview, select the arrow next to the condition.

[role="screenshot"]
image::images/log-threshold-alert-chart-previews.png[Log threshold chart previews]

The shaded area denotes the threshold that has been selected.

[discrete]
[[ratio-alerts]]
=== Ratio alerts

> By using a ratio configuration it's possible to be alerted when a ratio value meets a certain threshold. These configurations are useful when you want to know how one query (Query A) compares to the other query (Query B). The ratio threshold value is the result of the document count of Query A divided by the document count of Query B.

Reviewer: ```suggestion
To understand how one query compares to another query, create a ratio alert. This type of alert is triggered when a ratio value meets a specific threshold. The ratio threshold value is the document count of the first query (query A), divided by the document count of the second query (query B).
```

> The example below would trigger when there are more than twice as many error logs compared to warning logs.

Reviewer: ```suggestion
The following example triggers an alert when there are twice as many error logs to warning logs.
```

[role="screenshot"]
image::images/log-threshold-alert-ratio.png[Log threshold ratio alert]

[IMPORTANT]
=====

> If the document count of Query A or Query B is 0 this will result in an undefined / indeterminate ratio, as it's not possible to divide by 0. In this scenario no alert will trigger.

Reviewer: ```suggestion
As it is not possible to divide by 0, when the document count of query A or query B is 0, it results in an undefined/indeterminate ratio. In this scenario, no alert is triggered.
```
